W-15535654 Updates to improve SSO content jh (#32)
mulesoft/docs-api-experience-hub: modules/ROOT/pages/sso-use-case-2.adoc

> = Configuring SSO When the Identity Provider is Already Configured for Anypoint Platform

Reviewer: ```suggestion
= Configuring SSO When the Identity Provider Is Already Configured for Anypoint Platform
```

Reviewer: https://confluence.internal.salesforce.com/x/HQSGC

== Before You Begin
Ensure you have the following setup and context:

* An identity provider for Anypoint Platform is already configured.
* The user or the identity of the group of the users who have access to the application have been created in the identity provider. 

> * Read the information in the xref:enabling-sso-for-your-portal.adoc#before-you-begin [Before You Begin] section to understand the roles and permissions required and the information that API Experience Hub expects to configure custom user attributes and claims for the identity provider.

Reviewer: ```suggestion
* Understand the required roles, permissions, and information that API Experience Hub expects to configure custom user attributes and claims for the identity provider.
+
See xref:enabling-sso-for-your-portal.adoc#before-you-begin [Before You Begin].
```

Reviewer: (Maybe that changes the meaning....)

Is the "AEH expects" coming from the UI - or is this identity-provider terminology? It feels less than clear to me. I wonder if you could just say "requires" instead.

Author: yes, requires works

== Step 1: Enable SSO for Your Portal
In this step, you create and configure an application in the identity provider for the API Experience Hub portal, enable the ability to send group information in the application configuration, configure groups, and map the application to the identity provider for Anypoint Platform. Performing these steps require you to move back and forth between applications and the identity provider application to copy or add information. 

> === Create and Set Up An Application 

Reviewer: ```suggestion
=== Create and Set Up an Application 
```

Reviewer: Review the title capitalization guidelines.

Create a new application for the API Experience Hub portal in the identity provider. 

. In Okta, create an OpenID Connect Web application. For more information, see https://help.okta.com/oie/en-us/content/topics/apps/apps_app_integration_wizard_oidc.htm[Create OIDC app integrations^].
. From the *General Settings* section, complete the following fields:
+
[%header%autowidth.spread]
|===
|Field|Value
|*App integration name*|Enter a name for the app.
|*Grant type* |Select *Authorization Code*.
|*Assignments* |Select *Limit access to selected groups*.
|*Selected group(s)*|Enter the name of the group who must have access to the application for API Experience Hub and Anypoint Platform.
|===
. Verify that your identity provider sends the expected claims listed in the xref:enabling-sso-for-your-portal.adoc#before-you-begin[Before You Begin] section. 
. Configure the claims that the application sends.
+

> For example, for OpenID Connect, you can configure the groups claims using the following steps:

Reviewer: ```suggestion
For example, for OpenID Connect, configure groups claims using the following steps:
```

Reviewer: group claims?
a group claim?

Author: group claims.

> .. From the *Sign On* tab, click *Edit** from the *OpenID Connect ID Token* section.

Reviewer: ```suggestion
.. In the *Sign On* tab, *OpenID Connect ID Token* section, click *Edit*.
```

== Before You Begin
Ensure you have the following setup and context:

* An identity provider for Anypoint Platform is already configured.
* The user or the identity of the group of the users who have access to the application have been created in the identity provider. 
* Understand the required roles, permissions, and information that API Experience Hub requires to configure custom user attributes and claims for the identity provider.
+

> See xref:enabling-sso-for-your-portal.adoc#before-you-begin [Before You Begin].

Reviewer: Does this work with the extra space?

Author: Thanks for catching that. I will remove the extra space.

|*Selected group(s)*|Enter the name of the group who must have access to the application for API Experience Hub and Anypoint Platform.
|===
. Configure a custom groups claim filter that the application sends.

> +

Reviewer 2: Check my previous suggestion
```
. Configure required claims that the application sends.
+
Review the xref:enabling-sso-for-your-portal.adoc#before-you-begin[Before You Begin] section to verify that your Identity Provider send the expected claims.
For example, for OpenID in Okta you can configure the groups claim by doing the following:
```

Author: @nbattiato Should you review the Before You Begin section before the step in line 36 to Configure the required claims that the application sends?
